emloop/api.py

```diff
@@ -28,62 +28,51 @@ def create_output_dir(config: dict, output_root: str, default_model_name: str='U
     :param output_dir: if specified, create directory with output_dir path and return the path
     :return: path to the created output_dir
     """
-    if output_dir:
-        logging.info('\tOutput dir is: %s', output_dir)
+    if output_dir is not None:
+        logging.info('\tCreating an output dir with specified name: %s', output_dir)
         os.makedirs(output_dir, exist_ok=True)
-        create_config_log(config, output_dir)
-        return output_dir
 
-    logging.info('Creating output dir')
-
-    model_name = default_model_name
-    if 'name' not in config['model']:
-        logging.warning('\tmodel.name not found in config, defaulting to: %s', model_name)
     else:
-        model_name = config['model']['name']
-
-    if not os.path.exists(output_root):
-        logging.info('\tOutput root folder "%s" does not exist and will be created', output_root)
-        os.makedirs(output_root)
-
-    output_dir_format = os.environ.get('EMLOOP_OUTPUT_DIR_FORMAT', '{model_name}_{random_name}')
-    timestamp = datetime.now().strftime('%Y-%m-%d-%H-%M-%S')
-    random_name = get_random_name()
-    output_dir_name = output_dir_format.format(
-        timestamp=timestamp, model_name=model_name, random_name=random_name)
-
-    suffix = 0
-    while True:
-        if suffix > 0:
-            output_dir_name = output_dir_name + f'_{suffix}'
-
-        output_dir = os.path.join(output_root, output_dir_name)
-        try:
-            os.mkdir(output_dir)
-            break
-        except FileExistsError:
-            suffix += 1
-
-    create_config_log(config, output_dir)
-    logging.info(f'Created output directory with name {output_dir}')
-    return output_dir
-
-
-def create_config_log(config: dict, output_dir: str):
-    """
-    Dump the given config to YAML file under this dir
-    and register a file logger logging to a file under this dir
+        logging.info('Creating output dir')
```

Comment:
```suggestion
        logging.info('Generating output directory name')
```

Reply:
Hm.. so should I change it? For the quantization script I prefer the output_dir to specifie the whole path -> and generally I find it more practical like this, but if you find it confusing..